site/releases.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Apache BookKeeper Releases
+layout: community
+---
+
+## Download
+
+{{ site.stable_release }} is latest release. The current stable version is {{ site.stable_release }}.
+
+Releases are available to download from Apache mirrors: [Download](http://www.apache.org/dyn/closer.cgi/bookkeeper)
+
+You can verify your download by following these [procedures](http://www.apache.org/info/verification.html) and using these [KEYS](https://dist.apache.org/repos/dist/release/bookkeeper/KEYS).
+
+If you want to download older releases, they are available in the [Apache archive](http://archive.apache.org/dist/bookkeeper/).
+
+## Getting Started
+
+Once you've downloaded a BookKeeper release, instructions on getting up and running with a standalone cluster that you can run your laptop can be found
+in [Run BookKeeper locally]({{ site.baseurl }}/docs/getting-started/run-locally).
```

Comment:
When using `baseurl`, make sure *not* to include a slash, as this will break the link when you run it locally (when you run locally, the `baseurl` is `/`, which creates `//` URLs).

In this particular case it should be `{{ site.baseurl }}docs/getting-started/run-locally`.

Confusing and kind of annoying. Later on we can remove all of these `baseurl` links.